variables.tf

```
@@ -186,6 +186,24 @@ variable "auto_verified_attributes" {
   ]
 }
 
+variable "account_recovery_setting_recovery_mechanisms" {
```

I would suggest to keep this short as the context is also clear from the short name.

```suggestion
variable "account_recovery_mechanisms" {
```